app/test_split_integer.py

```
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(32, 6)) == 32
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(12, 6) == [2, 2, 2, 2, 2, 2]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(7, 1) == [7]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert split_integer(7, 4) == [1, 2, 2, 2]
```

As I mentioned in the previous review, the test `test_parts_should_be_sorted_when_they_are_not_equal` checks that the result of `split_integer(7, 4)` is equal to the sorted version of itself, but the function `split_integer` should return a sorted array by default, so sorting it in the test is not necessary and can mask errors in the implementation. Consider replacing `sorted(split_integer(7, 4))` with `[1, 1, 2, 3]`.